### DISCUSSION TITLE
join game event

### DISCUSSION THREAD

**Discussion by baihuamen** [FabricMC/fabric-api#4298] [Mod Dev Support] [answered]
### Description

I can't find a interface which will enable while player join game.Can someone help me？

**Comment by Cai-kunkun** [comment, marked as answer]
To do this,you should add a `onHudDrawer` like this.
Draw a "Hello world"text for eg:
```
HudRenderCallback.EVENT.register(((drawContext, tickCounter) -> {
    // Draw Hello World in full opacity white (255, 255, 255, 255) with no shadow
    drawContext.drawText(MinecraftClient.getInstance().textRenderer, "Hello World!", 2, 2, 0xFFFFFFFF, false);
}));
```
That will draw a text on the screen when the player was joined the game.
